include/UIKit/UIView.h

```diff
@@ -381,7 +381,9 @@ UIKIT_EXPORT_CLASS
 // WinObjC Xaml Extensions
 @class WXFrameworkElement;
```

Comment:
>WXFrameworkElement [](start = 7, length = 18)

Nit: All other forward declarations are done in a single pace in this file, except this one.